Agents/DerivationAgentPythonExample/README.md

```diff
@@ -0,0 +1,231 @@
+# Derivation Agent Python Example
+
+## Purpose
+This agent folder is intended as a template that one can copy and adapt to turn their own Python code into a derivation agent. The example requires [`pyderivationagent`](https://pypi.org/project/pyderivationagent/)>=1.4.1.
+
+This document covers four stages: development, test, package & publish, and deployment. For each stage, a step-by-step instruction is provided. Before continuing with this tutorial, it is recommended to read the [documentation](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB/python_derivation_agent) of `pyderivationagent` and all the relevant links.
+
+If you identified anything that can be improved to make it easier for newcomers, please feel free to open a [pull request](https://github.com/cambridge-cares/TheWorldAvatar/pulls) or get in touch with the maintainer of the package.
+
+
+&nbsp;
+## Environment setup
+
+For development and testing reasons, follow below instructions to get started.
+
+### Virtual environment setup
+
+It is highly recommended to install `pyderivationagent` packages in a [virtual environment](https://docs.python.org/3/tutorial/venv.html). The following steps can be taken to build a virtual environment:
```

Comment:
Potentially include that python >=3.8 is required here already to avoid possible issues later if `python` points to a version lower than required by setup.py

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/391/commits/c872c3b2a6e16f2080a545d71e9ce41d07873366